lib/config/rules/maintainability.js

```diff
@@ -0,0 +1,35 @@
+// Maintainability:
+// These rules make code maintenance easier
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Enforces a threshold of cyclomatic complexity.
+  'cyclomatic-complexity': false,
+  // Ensures the file ends with a newline.
+  eofline: true,
+  // Enforces indentation with tabs or spaces.
+  indent: [true, 'spaces'],
+  // Enforces a consistent linebreak style.
+  'linebreak-style': false,
+  // A file may not contain more than the specified number of classes
+  'max-classes-per-file': false,
+  // Requires files to remain under a certain number of lines
+  'max-file-line-count': false,
+  // Requires lines to be under a certain max length.
+  'max-line-length': false,
+  // Disallows default exports in ES6-style modules.
+  'no-default-export': false,
+  // Disallows mergeable namespaces in the same file. <FEEDBACK>
+  'no-mergeable-namespace': false,
```

Comment:
I don't see why you would want multiple namespaces that could be merged, so I'd do `true` here.